_posts/2013-01-14-alt-text.md

```diff
@@ -1,21 +1,29 @@
-- Use an empty `alt` attribute for any image that is decorative or not necessary for understanding the content of the page (`alt=""` or just `alt` - e.g. `<img src="prettyswirl.jpg" alt>`). If, for example, the information in a graph is explained in text below it, there's no need to duplicate that in `alt` text.
-- Images that are links should describe where they go. For example, a logo that is a link to your home page should have `alt="home page"`. But if the image is only one part of the contents of a link and the other contents (e.g. plain text) describes the link then use an empty `alt` attribute for the image - if you were to specify 'alt' text as well this would be redundant/repetitive.
+## A few tips on how and when to use the `alt` attribute:
+- Use the `alt` attribute for images that are used as more than decoration. The alternative text describes the primary purpose of an image and is rarely a detailed description of the image itself (unless, for example, the page is actually about a specific image).
+- Make sure the description of the image is useful. For example, if the image is of a company's logo, the `alt` should (often) be the company's name. The word "logo" is not necessary or useful as part of the alternative text.
+- Don't begin alternative text with "photo of..." or "picture of..."; assistive technologies already indicate the role of the element as an "image" or "graphic". Unnecessary redundancy makes for poor user experiences.
+- [Don't put line breaks in `alt` text](https://developer.paciellogroup.com/blog/2015/09/short-note-on-coding-alt-text/). It causes suboptimal effects when announced by screen readers.
+- Use an empty `alt` attribute for any image that is decorative or not necessary for understanding the content of the page (`alt=""` or just `alt` - e.g. `<img src="prettyswirl.jpg" alt>`). If, for example, the information in a graph is fully explained in text accompanying the image, there's no need to duplicate that in `alt` text.
+- Using an image several times in a website doesn't necessarily mean the `alt` attribute will be the same for each instance. For example, a logo within a website's header often doubles a link back to the home page. In this example, the `alt` would be useful as "Company's name - home". If that same logo were used in the footer of the site, next to the text "Company's name, copyright 20XX", then the logo would have an empty alt (`alt=""`) so as to not create a redundant announcement of the company's name.
+- There may be other times where images are used within links, but are decorative to the visible label of the link. For instance, if the image is an icon of a phone and part of a "contact" link, the `alt` could be empty.
```

Comment:
Use "For example," not "For instance,". You use instance in a more dev way in the above bullet point, which could muddy its meaning.